src/components/ServiceList.vue

```diff
@@ -153,7 +153,7 @@
         this.$dialog.confirm('Are you sure you want to delete the Service?', {okText: 'DELETE', cancelText: 'CANCEL'})
         .then(function () {
           // If Service is deployed, remove stack from Docker Swarm cluster as well
-          if (serviceEndpoint !== 'Not Deployed') {
+          if ((serviceEndpoint).toLowerCase() !== 'not deployed') {
```

Comment:
@taliaga It's more something that I changed in the API and caused this to not work properly, so I made it a bit less picky. I haven't set up a decent enough test suite for the UI because of how heavily it involves the API. Will think about it properly at some point.